plan/physical.go

```diff
@@ -273,7 +273,9 @@ func CreatePhysicalNode(id NodeID, spec PhysicalProcedureSpec) *PhysicalPlanNode
 	}
 }
 
-const NextPlanNodeIDKey = "NextPlanNodeID"
+type nodeIDKey string
+
+const NextPlanNodeIDKey nodeIDKey = "NextPlanNodeID"
```

Comment:
While you're here, can you change the casing on this so it is unexported?

`nextPlanNodeIDKey` instead.

Reply:
This is getting imported here https://github.com/influxdata/flux/blob/master/internal/spec/build.go#L49
so I will leave it as it is.